librz/core/cmd_descs/cmd_descs_generate.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python3
+# SPDX-FileCopyrightText: 2020-2021 ret2libc <sirmy15@gmail.com>
+# SPDX-License-Identifier: LGPL-3.0-only
 
 import argparse
 import os
 import sys
 
 import yaml
 
-CMDDESCS_C_TEMPLATE = """// SPDX-License-Identifier: LGPL-3.0-only
+CMDDESCS_C_TEMPLATE = """// SPDX-FileCopyrightText: 2020 RizinOrg <info@rizin.re>
```

Comment:
`SPDX-FileCopyrightText: 2020-{current_year} RizinOrg <info@rizin.re>` mabye

Reply:
I just didn't want to update this every year :D I don't think it's mandatory to have a range

Reply:
I put RIzinOrg instead of my name or the name of someone else to avoid having to adapt this line every time one adds a bunch of commands.